### ISSUE TITLE
Linking against brcmegl results in symbols conflicts

### ISSUE THREAD

**Issue by jpalus** [raspberrypi/firmware#1504]
**Describe the bug**
mpv fails to setup rpi video output with error: Could not get DISPMANX objects.

**To reproduce**
Build mpv linked against libbrcmegl.

**Expected behaviour**
mpv should work fine.

**Actual behaviour**
mpv fails.

**System**
* Pi2
* Arch
* $ /opt/vc/bin/vcgencmd version
Nov 18 2020 19:59:22 
Copyright (c) 2012 Broadcom
version 8e01026adc5a87d80f8748fc6a4fecb9012393cc (clean) (release) (start)
* Kernel 5.4.79

**Additional context**
brcmegl.pc file instructs to link against shared libbcm_host as well as static libvchostif. However those two both share common set of symbols, most importantly for mpv those for DISPMANX -- vc_dispman*. When linking both of them that creates a conflict and in turn a failure:
* bcm_host_init() when called is executed within context of libbcm_host and uses static context of this library
* mpv directly uses ie vc_dispamnx_display_open() which ends up linked with libvchostif (different static context)
* since bcm_host_init and vc_dispmanx_display_open are called in different context the latter returns 0 as if initialization was not made

In other words mpv binary lists bcm_host_init as undefined symbol satisfied by linking against dynamic library "libbcm_host", however it does not list "vc_dispmanx_display_open" as undefined since it was satisfied during static linking.

Either libvchostif should nod provide vc_dispman* symbols or brcmegl.pc should not instruct to link against libvchostif.
